website/dynamic-blocks/props-table.js

```diff
@@ -1,65 +1,155 @@
 /** @jsx jsx */
 import React, { Fragment } from 'react'; // Needed for within Keystone
-import { jsx } from '@westpac/core';
+import { jsx, useBrand } from '@westpac/core';
+import { Table, Thead, Tr, Th, Tbody, Td, Caption } from '@westpac/table';
+import { Heading } from '@westpac/heading';
 import PropTypes from '../../prop-types.json';
 
-const tableCellStyles = { border: 'solid 1px #e1ddd7', padding: 5 };
-const Th = ({ children }) => (
-	<th css={{ ...tableCellStyles, borderBottom: 'solid 3px #62194a' }}>{children}</th>
-);
-const Td = ({ children }) => <td css={{ ...tableCellStyles }}>{children}</td>;
-
-const getType = type => {
-	if (type.name === 'oneOfType') {
-		return type.value.map(t => t.name).join(', ');
+/**
+ * A small helper component for inline code blocks
+ */
+const Code = ({ children, ...rest }) => {
+	const { COLORS, SPACING } = useBrand();
+	return (
+		<pre
+			css={{
+				display: 'inline-block',
+				overflow: 'auto',
+				background: COLORS.background,
+				padding: `${SPACING(1, true)} ${SPACING(1)}`,
+				margin: `0 ${SPACING(1, true)}`,
+				border: `1px solid ${COLORS.border}`,
+				color: COLORS.hero,
+				fontSize: '0.9em',
+				borderRadius: '2px',
+			}}
+			{...rest}
+		>
+			<code>{children}</code>
+		</pre>
+	);
+};
+/**
+ * A small helper function to display values
+ *
+ * @param  {any} value - A value to be displayed
+ * @return {any}       - The value ready for string digestion
+ */
+function formatValue(value) {
+	if (typeof value === 'string') {
+		return `"${value}"`;
+	} else if (typeof value === 'boolean') {
+		return value ? 'true' : 'false';
+	} else {
+		return value;
 	}
-
-	if (type.name === 'shape') {
-		return 'Object';
+}
+/**
```

Comment:
I'd love some room for breathing
```suggestion
}

/**
```